das/avail/utils.go

```diff
@@ -0,0 +1,126 @@
+package avail
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	gsrpc "github.com/centrifuge/go-substrate-rpc-client/v4"
+	gsrpc_types "github.com/centrifuge/go-substrate-rpc-client/v4/types"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/vedhavyas/go-subkey"
+)
+
+var localNonce uint32 = 0
+
+func GetAccountNonce(accountNonce uint32) uint32 {
```

Comment:
remove this

Reply:
Yes, take care of in 154e91e6633633c9ab4fa3227dcfb293e6c8e37f